dbc2val/overlay/README.md

```diff
@@ -0,0 +1,219 @@
+# Using overlays as alternative mapping
+
+## Example 1 - Boolean
+
+```
+VCFRONT_passengerPresent:
+  vss: # taken from https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/SingleSeat.vspec
+    datatype: boolean 
+    type: sensor
+    unit: none
+    description: Does the seat have a passenger in it.
+  databroker:
+    datatype: 1 # BOOL number taken from types_pb2.py
+    changetype: 2 # ON_CHANGE number taken from types_pb2.py
+  minupdatedelay: 1000
+  targets:
+    Vehicle.Cabin.Seat.Row1.Pos2.IsOccupied:
+      transform:
+        fullmapping:
+          1: "true"
+          0: "false"
+```
+
+A functionally equal mapping could possibly be like this:
+
+```
+Vehicle.Cabin.Seat.Row1.Pos2.IsOccupied:
```

Comment:
@SebastianSchildt - as of today we cannot support this type of overlays, see https://github.com/COVESA/vss-tools/issues/201